primitives/core/src/traits.rs

```diff
@@ -195,6 +195,48 @@ pub trait BareCryptoStore: Send + Sync {
 	) -> Result<VRFSignature, Error>;
 }
 
+/// Something that generates, stores and provides access to keys.
+pub trait CryptoExt: Send + Sync {
```

Comment:
What is with the batching stuff, why isn't that here? Where do we draw the line?

Reply:
Good catch, we should defintely move all crypto stuff here to make it consistent.

Reply:
batching eventually calls methods on `CryptoExt`